readme.md

```diff
@@ -154,6 +154,38 @@ const successes = await ley.up({ ... });
 
 A config file is entirely optional since `ley` assumes that you're providing the correct environment variable(s) for your client driver. However, that may not always be possible. In those instances, a `ley.config.js` file (default file name) can be used to adjust your [driver](#drivers)'s `connect` method – the file contents are passed directly to this function.
 
+For example, if your hosting provider sets non-standard environment variables for the client driver (like Heroku does), you could extract the information and set the standard environment variables:
+
+```js
+// ley.config.js
+if (process.env.DATABASE_URL) {
+  const url = require('url');
+
+  // Extract the connection information from the Heroku environment variable
```

Comment:
Thanks!

I generally prefer low-level demonstrations like this (for clarity) but given that connection strings can be pretty _wild_, do you think it'd make sense to link to [`pg-connection-string`](https://www.npmjs.com/package/pg-connection-string) directly for the "official" example?

Could also mention that `pg` users/drivers can define a `connectionString` directly, skipping the need to manually parse: https://node-postgres.com/features/connecting#connection-uri

Reply:
Ok, switched to `pg-connection-string`.

I was thinking of also including a comment about `pg`, but I think that will muddle this specific example, because then it will be about using `module.exports` again... what do you think?

Reply:
Meh, I think this is fine. Users of `pg` should be able to find the `connectionString` option on their own. But even if they don't, this configuration works across the board.